### DISCUSSION TITLE
How to do lfg from 2 more people.

### DISCUSSION THREAD

**Discussion by reactmay** [azerothcore/azerothcore-wotlk#12838] [General]
Hi all!

There is a SoloLfg module, but I need to know where I can change the value so that the dungeon search works for example from 2-3 players (the role is not important). It is necessary that the search for dungeons work only for at least 2-3 players, please tell me.

**Comment by LukasVolgger** [comment]
https://github.com/azerothcore/mod-solo-lfg

**Reply by reactmay** [reply]
I know this is from one person, but if I want to make it from two or three people, so that players cannot go solo, I need from two or three players.

**Reply by LukasVolgger** [reply]
Afaik you can join lfg with that module no matter how big the group size os. Maybe you take a look at the worldserver.conf? Maybe you find there something

**Comment by LukasVolgger** [comment]
You can also use `.debug lfg` as gm
